script.js

```diff
@@ -0,0 +1,354 @@
+const a = [];
```

Comment:
Why do you split code for each row assignment?
```suggestion
const a = [
    [
        {className:"Backquote",eng:{caseDown:"`",caseUp:"~"},rus:{caseDown:"ё",caseUp:"Ё"}},
        // ... items omitted
    ],
    // ... rows omitted
];
```